pkg/band/as_923_rp2_v1_0_3.go

```diff
@@ -0,0 +1,124 @@
+// Copyright © 2021 The Things Network Foundation, The Things Industries B.V.
```

Comment:
I guess this is mostly copy-paste from existing files, but let's use the current year.